install/tools/man/ipa-adtrust-install.1

```diff
@@ -76,7 +76,7 @@ change the name. Please note that changing the NetBIOS name might break
 existing trust relationships to other domains.
 .TP
 \fB\-\-add\-sids\fR
-Add SIDs to existing users and groups as on of final steps of the
+Add SIDs to existing users and groups as one of the final steps of
```

Comment:
The "the" should have stayed there, it's related to the "run" rather than ipa-adtrust-install.